index.html

```diff
@@ -215,32 +215,65 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
-          <ul>
-            <li>When a <a data-cite="html/forms.html#category-label">labelable element</a> is referenced by a `label` element's `for` attribute, or a descendant of a `label` element, the labelable element's UIA `LabeledBy` property points to the UIA element for the `label` element.</li>
-            <li>Elements mapped to the `Text` Control Type are not generally represented as <a class="termref" data-lt="accessible object">accessible objects</a> in the <a class="termref">accessibility tree</a>, but are just part of the `Text` Control Pattern implemented for the whole HTML document. However, if they have any `aria-` attributes or an explicit `tabindex` specified, elements mapped to the `Text` Control Type will be represented as <a class="termref" data-lt="accessible object">accessible objects</a> in the <a class="termref">accessibility tree</a>.</li>
-          </ul>
+          the author has not specified an explicit role to the element, 
+        <li>
+          and the author has specified properties which require a minimum role mapping for the element.
         </li>
+      </ul>
+      <p>
+        The properties which will require a minimum role mapping are specific global HTML attributes, identified in the <a href="#attribute-mapping-table">Mappings of HTML attributes table</a>.
+      </p>
+      <p>
+        When these conditions are met, the browser MUST expose an object using the mappings defined in CORE-AAM for the specified minimum role.
+      </p>
+    </section>
+    <section>
+      <h3>HTML Element Role Mappings</h3>
+      <ul>
+        <li>HTML elements with implicit WAI-ARIA role semantics MUST be mapped to platform <a class="termref">accessibility APIs</a> according to the identified WAI-ARIA role mapping as defined in the [[core-aam-1.2]] specification.</li>
+        <li>"Not mapped" means the element is not exposed via an <a class="termref">accessibility API</a>. Commonly, an element is not mapped because it's not displayed as part of the user interface. However, authors can force some of these elements to be rendered. For instance, by overriding user agent styles to render elements that would have been otherwise set to `display: none`. In these cases, the user agent SHOULD map such elements to the role of <a class="core-mapping" href="#role-map-generic">`generic`</a>, unless other HTML features have been specified which would require a more specific <a>minimum role</a> to be exposed.</li>
+        <li>Where an element is indicated as having <q>No corresponding (WAI-ARIA) role</q>, or is mapped to the <a class="core-mapping" href="#role-map-generic">`generic`</a> role, user agents MUST NOT expose the <a class="core-mapping" href="#ariaRoleDescription">`aria-roledescription`</a> property value in the <a class="termref">accessibility tree</a> unless the element has an explicit, conforming `role` attribute value which [[WAI-ARIA-1.2]] does not prohibit the use of `aria-roledescription` be used with.</li>
```

Comment:
```suggestion
        <li>Where an element is indicated as having <q>No corresponding (WAI-ARIA) role</q>, or is mapped to the <a class="core-mapping" href="#role-map-generic">`generic`</a> role, user agents MUST NOT expose the <a class="core-mapping" href="#ariaRoleDescription">`aria-roledescription`</a> property value in the <a class="termref">accessibility tree</a> unless the element has an explicit, conforming `role` attribute value for which [[WAI-ARIA-1.2]] does not prohibit the use of `aria-roledescription`.</li>
```

I think this tweak preserves the meaning and is more quickly understood?